Dockerfile.openshift

```diff
@@ -36,3 +36,6 @@ RUN chown frr:frr /etc/frr/daemons /etc/frr/frr.conf
 
 WORKDIR /root
 ENTRYPOINT ["/root/frr.sh", "frr-node"]
+
```

Comment:
do we need this too 
```
# Simple init manager for reaping processes and forwarding signals
ENTRYPOINT ["/sbin/tini", "--"]
```

Reply:
Yep I think so, let's try to adhere to the upstream config, replacing the current entrypoint with the other one.

Reply:
We don't need that frr.sh script anymore? And btw, are we going to use the same command as here: https://github.com/metallb/metallb/blob/main/manifests/metallb-frr.yaml#L493? If so that would override the entrypoint we define here (if we need it or not).

Reply:
I'd keep it aligned to u/s anyway, in case we change something in the command in the operator.